iree/tools/iree-benchmark-module-main.cc

```diff
@@ -266,44 +271,12 @@ class IREEBenchmark {
 int main(int argc, char** argv) {
   IREE_TRACE_SCOPE0("main");
 
-  // We have to contend with two flag parsing libraries here: absl's and
-  // benchmark's. To make matters worse, both define the `--help` flag. To
-  // ensure that each is able to parse its own flags, we use an absl "internal"
-  // function (still with public visibility) to parse while ignoring undefined
-  // flags. If it sees `--help` it will exit here, so we include the benchmark
-  // library usage information in the manually-set help output. Then we let
-  // benchmark parse its flags. Finally we call the normal initialization
-  // function to do other IREE initialization including flag parsing with
-  // normal options. Any remaining flags will be unknown and result in an error.
-  absl::SetProgramUsageMessage(
-      "iree-benchmark-module \n"
-      "    --module_file=module.vmfb\n"
-      "    --entry_function=exported_function_to_benchmark\n"
-      "      If this is not set, all the exported functions will be \n"
-      "      benchmarked and they are expected to not have input arguments\n"
-      "    [--function_inputs=2xi32=1 2,1x2xf32=2 1 | \n"
-      "     --function_inputs_file=file_with_function_inputs]\n"
-      "    [--driver=vmla]\n"
-      "\n\n"
-      "  Optional flags from third_party/benchmark/src/benchmark.cc:\n"
-      "    [--benchmark_list_tests={true|false}]\n"
-      "    [--benchmark_filter=<regex>]\n"
-      "    [--benchmark_min_time=<min_time>]\n"
-      "    [--benchmark_repetitions=<num_repetitions>]\n"
-      "    [--benchmark_report_aggregates_only={true|false}]\n"
-      "    [--benchmark_display_aggregates_only={true|false}]\n"
-      "    [--benchmark_format=<console|json|csv>]\n"
-      "    [--benchmark_out=<filename>]\n"
-      "    [--benchmark_out_format=<json|console|csv>]\n"
-      "    [--benchmark_color={auto|true|false}]\n"
-      "    [--benchmark_counters_tabular={true|false}]\n"
-      "    [--v=<verbosity>]\n");
-  absl::flags_internal::ParseCommandLineImpl(
-      argc, argv, absl::flags_internal::ArgvListAction::kRemoveParsedArgs,
-      absl::flags_internal::UsageFlagsAction::kHandleUsage,
-      absl::flags_internal::OnUndefinedFlag::kIgnoreUndefined);
+  // Pass through flags to benchmark (allowing --help to fall through).
```

Comment:
:tada: Oh thank god we can get rid of this. I was just waiting until it came back to bite us in some way